sympy/physics/continuum_mechanics/cable.py

```diff
@@ -585,3 +588,191 @@ def solve(self, *args):
+                else:
+                    y = ((sorted_position[i][1][1] - sorted_position[i-1][1][1] )*(x-sorted_position[i-1][1][0]))/(sorted_position[i][1][0]- sorted_position[i-1][1][0]) + sorted_position[i-1][1][1]
+                line_func.append((y,x<=sorted_position[i][1][0]))
+
+            y = ((sorted_position[len(sorted_position)-1][1][1] - self._right_support[1])*(x-self._right_support[0]))/(sorted_position[i][1][0]- self._right_support[0]) + self._right_support[1]
+            line_func.append((y,x<=self._right_support[0]))
+            return [Piecewise(*line_func)]
+
+        elif order == 0:
+            x0 = self._lowest_x_global
+            diff_force_height = max_diff*0.075
+
+            a,c,x,y = symbols('a c x y')
+            parabola_eqn = a*(x-x0)**2 + c - y
+
+            points = [(self._left_support[0],self._left_support[1]),(self._right_support[0],self._right_support[1])]
+            equations = []
+            for px, py in points:
+                equations.append(parabola_eqn.subs({x: px, y: py}))
+            solution = solve(equations, (a, c))
+            parabola_eqn = solution[a]*(x-x0)**2 + solution[c]
+            return [parabola_eqn, self._lowest_y_global - diff_force_height]
+
+    def _draw_loads(self,order):
+        xy_min = min(self._left_support[0],self._lowest_y_global)
+        xy_max = max(self._right_support[0], max(self._right_support[1],self._left_support[1]))
+        max_diff = xy_max - xy_min
+        if(order==-1):
+            arrow_length = max_diff*0.1
+            force_arrows = []
+            for key in self._loads['point_load']:
+                force_arrows.append(
+                    {
+                        'text': '',
+                        'xy':(self._loads_position[key][0]+arrow_length*cos(rad(self._loads['point_load'][key][1])),\
+                              self._loads_position[key][1] + arrow_length*sin(rad(self._loads['point_load'][key][1]))),
+                        'xytext': (self._loads_position[key][0],self._loads_position[key][1]),
+                        'arrowprops': {'width': 1, 'headlength':3, 'headwidth':3 , 'facecolor': 'black', }
+                    }
+                )
+                mag = self._loads['point_load'][key][0]
+                force_arrows.append(
+                    {
+                        'text':f'{mag}N',
+                        'xy': (self._loads_position[key][0]+arrow_length*1.6*cos(rad(self._loads['point_load'][key][1])),\
+                               self._loads_position[key][1] + arrow_length*1.6*sin(rad(self._loads['point_load'][key][1]))),
+                    }
+                )
+            return force_arrows
+
+        elif (order == 0):
+            x = symbols('x')
+            force_arrows = []
+            x_val = [self._left_support[0] + ((self._right_support[0]-self._left_support[0])/10)*i for i in range(1,10)]
+            for i in x_val:
+                force_arrows.append(
```

Comment:
Here I have added two distributed loads of different magnitudes. I think these should be added instead of plotting on the same plot separately. Also, despite different magnitudes the arrow lengths are the same. 

![Figure_7](https://github.com/sympy/sympy/assets/92469698/51a4814f-cdb9-45e4-aefc-6947796f0928)

Reply:
Makes sense. If you will be adding the distributed loads then there is no point in discussing about arrow lengths because you won't be having multiple distributed loads on a cable.

Reply:
```
from sympy.physics.continuum_mechanics.cable import Cable
c=Cable(("A", 0, 40),("B", 100, 20))
c.apply_load(0, ("X", 850))
c.apply_load(0,('Y',200))
c.solve(58.58)
```
gives the following graph
![image](https://github.com/sympy/sympy/assets/138311586/f630733b-1879-4edb-b615-a5b7bb61c0d5)